.changeset/pink-rocks-bake.md

```diff
@@ -0,0 +1,6 @@
+---
+"@razorpay/blade": patch
+---
+
+feat(TextArea): Tagged Inputs
```

Comment:
We have also added this to TextInput right?

Lets format the changelog a bit better